config/default.json

```diff
@@ -19,6 +19,7 @@
       { "period": "5 minutes", "limit": 0 }
     ]
   },
+  "jsonlogging": "true",
```

Comment:
Bike-shedding here, but let's call this `mozlog` so it specifically identifies the output format that all our data processing pipelines rely on.